spec/controllers/admin/customers_controller_spec.rb

```diff
@@ -37,6 +37,35 @@
       locals = capture_view_locals { get :index }
       expect(locals[:show_search_bar]).to be_truthy
     end
+
+    it "sorts by id by default" do
+      customer1 = create(:customer)
+      customer2 = create(:customer)
+      customers = [customer1, customer2]
+
+      locals = capture_view_locals { get :index }
+      expect(locals[:resources].map(&:id)).to eq customers.map(&:id).sort
+    end
+
+    context "with alternate sorting attributes" do
+      controller(Admin::CustomersController) do
+        def default_sorting_attribute
+          :name
+        end
+
+        def default_sorting_direction
+          :desc
+        end
+      end
+
+      it "retreives resources in the correct order" do
```

Comment:
In fact we can use the new "suggestion" feature :-)

```suggestion
      it "retrieves resources in the correct order" do
```